test/README.md

```diff
@@ -160,15 +160,16 @@ New test images should be placed in `./test/test_images`.
 These flags are useful for running against an existing cluster, making use of
 your existing [environment setup](../DEVELOPMENT.md#setup-your-environment).
 
-Tests importing [`github.com/knative/serving/test`](#test-library) recognize
+Tests importing [`knative.dev/serving/test`](#test-library) recognize
 these flags:
 
-- [All flags added by `knative/pkg/test`](https://github.com/knative/pkg/tree/master/test#flags)
-- [`--dockerrepo`](#overriding-docker-repo)
-- [`--tag`](#using-a-docker-tag)
-- [`--ingressendpoint`](#using-a-custom-ingress-endpoint)
+- [All flags added by `knative/pkg/test`](https://github.com/knative/pkg/tree/master/test#flags) such as:
+  - [`--dockerrepo`](#overriding-docker-repo)
```

Comment:
Should these be on the same indentation level as the last three flags?

Reply:
The `--dockerrepo`, `--tag`, `--ingressendpoint` belong to `knative/pkg/test`. They are public flag for all test. 
The last three flags are special flag for `knative/serving/test`.

Reply:
Ah got it. Thanks for the clarification!